kiwi/builder/live.py

```diff
@@ -241,8 +241,8 @@ def create(self):
         # calculate size and decide if we need UDF
         if rootsize.accumulate_mbyte_file_sizes() > 4096:
             log.info('ISO exceeds 4G size, using UDF filesystem')
-            custom_iso_args['create_options'].append('-allow-limited-size')
-            custom_iso_args['create_options'].append('-udf')
+            custom_iso_args['create_options'].append('-iso-level')
+            custom_iso_args['create_options'].append('3')
```

Comment:
Apparently it works fine using -udf in genisoimage and in mkisofs. In both cases I could boot a virtual machine from the generated ISO and I could also boot from a USB device using that ISO.